frontends/web/src/components/layout/header.module.css

```diff
@@ -1,3 +1,25 @@
+.children {
```

Comment:
oups misunderstanding. I meant sort the CSS properties. so that `align-items` is first and `z-index` last, that way it would be easy to spot that there are 2x `padding` in there.